plugins/yara/functional-test/tests/tests.bats

```diff
@@ -0,0 +1,24 @@
+# shellcheck disable=SC2016
+
+: "${BATS_TEST_TIMEOUT:=120}"
+
+RULE_DIR="$BATS_SUITE_DIRNAME"
+
+setup() {
+  bats_load_library bats-support
+  bats_load_library bats-assert
+  bats_load_library bats-tenzir
+}
+
+@test "generate matches for source rule" {
+  echo 'foo barbar baz' | check tenzir "load stdin | yara ${RULE_DIR}/test.yara"
+}
+
+@test "generate matches for compiled rule" {
+  yarac ${RULE_DIR}/test.yara test.yarac
```

Comment:
@tobim invoking `yarac` compiles the rule and generates a file `test.yarac` in the current directory. Is there the equivalent of a "cwd" for bats? When I run it in the source tree, it ends up there. I guess I'm looking for the equivalent of `CMAKE_CURRENT_BINARY_DIRECTORY` or so.

Reply:
There is `BATS_TEST_DIRNAME`, maybe that helps?

Reply:
Or maybe you actually want to use `TENZIR_DB_DIRECTORY` for that one. There is no higher level test runtime state dir at the moment.

Reply:
Can you take a look at 55b64a8baf? I used `BATS_TEST_DIRNAME` here, but that puts generated file into the source tree. `TENZIR_DB_DIRECTORY` feels weird. Maybe `mktemp`?